shared/utils/generate-from-templates.py

```diff
@@ -118,41 +119,46 @@ def _get_csv_dir(self):
         return self.csv_dirs[self.current_oval]
 
     def _get_template_dir(self):
+        # FIXME: _get_template_dir already defined a few lines above!!
```

Comment:
MG, stupid dynamic languages.

This implementation is ok if we don't use versions of OVAL - which is used as I know only in Debian. And unfortunately the issue caused change only in generated OVAL, but I've checked only remediations and not oval. So I haven't noticed it.

Shame on me. I will open a short PR.